src/QMCHamiltonians/QMCHamiltonian.cpp

```diff
@@ -491,8 +490,9 @@ QMCHamiltonian::Return_t QMCHamiltonian::evaluate(ParticleSet& P)
 std::vector<QMCHamiltonian::RealType> QMCHamiltonian::flex_evaluate(const RefVector<QMCHamiltonian>& H_list,
                                    const RefVector<ParticleSet>& P_list)
 {
-  // if (H_list.size() > 1)
-  // {
+  std::vector<RealType> local_energies(H_list.size(),0.0);
```

Comment:
Although I hate QMCHamiltonian::Return_t, Please use consistent data type as single evaluation.
Return_t is always on full precision. We want local energy not losing accuracy.